lte/gateway/fabfile.py

```diff
@@ -208,10 +209,16 @@ def integ_test(gateway_host=None, test_host=None, trf_host=None,
     """
 
     destroy_vm = bool(strtobool(destroy_vm))
+    provision_magma = bool(strtobool(provision_magma))
+
+    if not destroy_vm and provision_magma:
+        # this will only run in case we don't destroy the VM
+        os.system("vagrant provision magma")
```

Comment:
I would use `local("vagrant provision magma")` this way you don't have to include `os` module

Reply:
ohh was looking for that!!
Thanks!